eng/validation/test-publishing.ps1

```diff
@@ -17,12 +18,11 @@ $darc = & "$PSScriptRoot\get-darc.ps1"
 $global:buildId = $buildId
 $global:targetChannel = "General Testing"
 $global:azdoToken = $azdoToken
-$global:githubUser = $githubUser
+$global:azdoUser = $azdoUser
+$global:azdoOrg = $azdoOrg
+$global:azdoProject = $azdoProject
 $global:githubPAT = $githubPAT
```

Comment:
If we're ripping GitHub out of the equation here, I wonder if darc even needs the githubPAT anymore

Reply:
I left that bit in there because there are circumstances where it falls back for getting VersionDetails.xml and it didn't super matter.